houston/types.go

```diff
@@ -301,14 +310,14 @@ type FeatureFlags struct {
 }
 
 // coerce a string into SemVer if possible
-func coerce(version string) *semver.Version {
+func coerce(version string) (*semver.Version, error) {
 	v, err := semver.NewVersion(version)
 	if err != nil {
-		fmt.Println(err)
+		return nil, err
 	}
 	coerceVer, err := semver.NewVersion(fmt.Sprintf("%d.%d.%d", v.Major(), v.Minor(), v.Patch()))
 	if err != nil {
 		fmt.Println(err)
```

Comment:
nitpicking: should we return the error from here as well, just to be consistent and safe, else there is a possibility of `coerce` returning `nil, nil`

Reply:
You're right, for some reason I didn't change this line O_O